src/pydata_sphinx_theme/__init__.py

```diff
@@ -1084,6 +1085,43 @@ def setup_translators(app):
             app.set_translator(name, translator, override=True)
 
 
+# ------------------------------------------------------------------------------
+# customize method for logo management
+# ------------------------------------------------------------------------------
+
+
+def setup_logo_path(
+    app: Sphinx, pagename: str, templatename: str, context: dict, doctree: nodes.Node
+) -> None:
+    """Set up relative paths to logos independant from the version"""
+
+    # get informations from the context
+    pathto = context.get("pathto")
+    logo = context.get("logo")
+    image_light = context.get("theme_logo", {}).get("image_light")
+    image_dark = context.get("theme_logo", {}).get("image_dark")
+
+    # create theme_options["logo"] if not existing
+    if "theme_logo" not in context:
+        context["theme_logo"] = {}
+
+    # create the logo_url
+    if logo and not isurl(logo):
+        logo = pathto(f"_static/{logo}", resource=True)
```

Comment:
I'm a little confused - won't this result in the same `_static/_static` problem since we're defining `html_logo` to start with `_static` above?

Reply:
I also changed the html template so _static will be written only once. 
I checked the [documentation](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_logo) and html_logo need to be set from the conf folder. That being said the `logo` variable is only including the stem as the file is copied in the `_static` destination folder.